lib/feature_management.rb

```diff
@@ -116,27 +116,13 @@ def self.log_to_stdout?
     !Rails.env.test? && IdentityConfig.store.log_to_stdout
   end
 
-  # Whether or not we can call the phone_info endpoint at all
-  def self.voip_check?
-    AppConfig.env.voip_check == 'true'
-  end
-
-  # Whether or not we should block VOIP phone numbers
-  def self.voip_block?
-    AppConfig.env.voip_block == 'true'
-  end
-
   def self.ruby_workers_enabled?
     AppConfig.env.ruby_workers_enabled == 'true'
   end
 
   # Manual allowlist for VOIPs, should only include known VOIPs that we use for smoke tests
   # @return [Set<String>] set of phone numbers normalized to e164
   def self.voip_allowed_phones
-    @voip_allowed_phones ||= if (allowed_phones = AppConfig.env.voip_allowed_phones).present?
-      JSON.parse(allowed_phones).map { |p| Phonelib.parse(p).e164 }.to_set
```

Comment:
where did the e164 normalization go?

Reply:
whoops

Reply:
Missed that this was doing that when I was going through these. Just put it back. Good catch!